tomviz/DockerExecutor.h

```diff
@@ -0,0 +1,59 @@
+/* This source file is part of the Tomviz project, https://tomviz.org/.
+   It is released under the 3-Clause BSD License, see "LICENSE". */
+
+#ifndef tomvizDockerExecutor_h
+#define tomvizDockerExecutor_h
+
+#include <QObject>
+
+#include "PipelineExecutor.h"
+
+namespace tomviz {
+class DataSource;
+class Operator;
+class Pipeline;
+
+namespace docker {
+class DockerStopInvocation;
+class DockerRunInvocation;
+}
+
```

Comment:
General comment, but I think it is worth spending a few moments on. Can we add one or two sentences for the new classes describing their high level purpose. I think here it is:

Orchestrate the execution of pipelines in Docker containers, providing a pristine container-based pipeline environment.

Or something like that, I may have misunderstood, and then it would be good to get your take on it.

Reply:
Done